src/java/picard/vcf/GA4GHSchemeWithMissingAsNoCall.java

```diff
@@ -0,0 +1,47 @@
+package picard.vcf;
+
+import picard.PicardException;
+import picard.vcf.GenotypeConcordanceStates.*;
+
+/**
+ * The scheme is defined in the constructor.
```

Comment:
Most of these comments are redundant (see parent class).  Also, we need to add some language to differentiate between this scheme and its parent's scheme.